website/docs/docs/deploy/continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Continuous deployment in dbt Cloud"
+sidebar_label: "Continuous deployment"
+description: "Learn about continuous deployment (CD) workflows "
+---
+
+To help you improve data transformations and to ensure up-to-date data products, you can run [merge jobs](/docs/deploy/merge-jobs) to implement a continuous deployment (CD) workflow in dbt Cloud. Merge jobs can automatically build modified models for you whenever a pull request (PR) merges, making sure the latest changes are in production (such as modifications to code that generates the tables) &mdash; no waiting for the next scheduled job to run. 
+
+You can also implement continuous integration (CI) in dbt Cloud, which can help further to reduce the time it takes to push changes to production and improve code quality. To learn more, refer to [Continuous integration in dbt Cloud](/docs/deploy/continuous-integration). 
+
```

Comment:
i think it would be helpful to add a diagram to the above section

Reply:
log: filed [asana ticket](https://app.asana.com/0/1202263237307725/1207890130669766/f) for diagram request